cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,286 @@
+import datetime
+
+from django.db.models import Count, F
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+
+from cinema.models import Movie, MovieSession, Actor, Genre, CinemaHall
+from cinema.serializers import MovieListSerializer, MovieSessionListSerializer
+from user.models import User
+
+MOVIE_LIST_URL = reverse("cinema:movie-list")
+MOVIE_SESSION_LIST_URL = reverse("cinema:moviesession-list")
+
+
+class MovieAndMovieSessionTestCaseForAnonimUser(TestCase):
+    def setUp(self):
+        self.client = APIClient()
```

Comment:
You are creating a new APIClient instance in each test case. Consider creating a single instance at the module level to avoid redundancy.